CMakeLists.txt

```diff
@@ -264,6 +265,9 @@ endif()
 
 if(ENABLE_OPENACC)
     set(COMPILE_LIBRARY_TYPE "STATIC")
+    if (ENABLE_UNIFIED)
+        add_definitions( -DUNIFIED_MEMORY)
```

Comment:
I haven't seen a usage of this definition anywhere in the code. Could it be removed? Maybe I am missing something

Reply:
If you check the `nrn_acc_managed.cpp` you'll see that the `UNIFIED_MEMORY` clause is used to select the type of allocation, either Unified or Explicit Copy.